repos/system_upgrade/el7toel8/actors/persistentnetnamesdisable/tests/test_persistentnetnamesdisable.py

```diff
@@ -0,0 +1,33 @@
+from leapp.snactor.fixture import current_actor_context
+from leapp.reporting import Report
+from leapp.models import KernelCmdlineArg, PersistentNetNamesFacts, Interface, PCIAddress
+
+
+def test_actor_single_eth0(current_actor_context):
+    pci = PCIAddress(domain="ax", bus="bx", function="cx", device="dx")
+    interface = [Interface(name="eth0", mac="52:54:00:0b:4a:6d", vendor="adh",
+                           driver="as", devpath="rr", pci_info=pci)]
+    current_actor_context.feed(PersistentNetNamesFacts(interfaces=interface))
+    current_actor_context.run()
+    assert not current_actor_context.consume(Report)
+
+
+def test_actor_with_more_ethX(current_actor_context):
+    pci1 = PCIAddress(domain="ax", bus="bx", function="cx", device="dx")
+    pci2 = PCIAddress(domain="ay", bus="by", function="cy", device="dy")
+    interface = [Interface(name="eth0", mac="52:54:00:0b:4a:6d", vendor="adh",
+                           driver="as", devpath="rrx", pci_info=pci1),
+                 Interface(name="eth1", mac="52:54:00:0b:4a:6a", vendor="adh",
+                           driver="az", devpath="rry", pci_info=pci2)]
+    current_actor_context.feed(PersistentNetNamesFacts(interfaces=interface))
+    current_actor_context.run()
+    assert 'inhibitor' in current_actor_context.consume(Report)[0].flags
+
```

Comment:
missing test 1 eth & 1+ non-eth

Reply:
@kokesak Do not forget to "resolve conversation" when you resolve the requested changes.